firebase-functions/src/main/java/com/google/firebase/functions/HttpsCallableReference.kt

```diff
@@ -125,6 +126,56 @@ public class HttpsCallableReference {
     }
   }
 
+  /**
+   * Streams data to the specified HTTPS endpoint.
+   *
+   * The data passed into the trigger can be any of the following types:
+   *
+   * * Any primitive type, including null, int, long, float, and boolean.
+   * * [String]
+   * * [List&amp;lt;?&amp;gt;][java.util.List], where the contained objects are also one of these
```

Comment:
Will `[List&amp;lt;?&amp;gt;]` appear in properly formatted way in Android studio?

Reply:
It does not look good on the 'call' method either. Do we need to change both call and stream?

Reply:
Yeah, these look like html entities. I'm guessing they can changed for `<` and `>`. cc: @emilypgoogle 
```suggestion
   * * [List&amp;lt;?&amp;gt;][java.util.List], where the contained objects are also one of these
```

Reply:
I'll take a look after merging